### PR TITLE
MMI-3116 Update Source and Series CBRA source

### PR DESCRIPTION
**Pull request by laidaoyu**
- added is CBRA source to source and series page
- updated source and series data models

### CODE REVIEW

File: libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+
+ALTER TABLE public."source" DROP COLUMN "is_cbra_source";
```
Comment (Fosol) [inline libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql:4]: Why do you need these?  EF migration should handle all of this for you.

File: libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+
+ALTER TABLE public."source" ADD COLUMN "is_cbra_source" boolean NOT NULL DEFAULT false;
```
Comment (Fosol) [inline libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql:4]: Why do you need these?  EF migration should handle all of this for you.

File: libs/net/dal/Configuration/SeriesConfiguration.cs [hunk 3]
```diff
@@ -12,6 +12,7 @@ public override void Configure(EntityTypeBuilder<Series> builder)
         builder.Property(m => m.SourceId).IsRequired(false);
         builder.Property(m => m.AutoTranscribe).IsRequired();
         builder.Property(m => m.UseInTopics).IsRequired();
+        builder.Property(m => m.IsCBRASource).IsRequired(false);
```
Comment (Fosol) [inline libs/net/dal/Configuration/SeriesConfiguration.cs:15]: According to your entity it is required.

File: libs/net/dal/Configuration/SourceConfiguration.cs [hunk 4]
```diff
@@ -21,6 +21,7 @@ public override void Configure(EntityTypeBuilder<Source> builder)
         builder.Property(m => m.DisableTranscribe).IsRequired();
         builder.Property(m => m.UseInTopics).IsRequired();
         builder.Property(m => m.Configuration).IsRequired().HasColumnType("jsonb").HasDefaultValueSql("'{}'::jsonb");
+        builder.Property(m => m.IsCBRASource).IsRequired(false);
```
Comment (Fosol) [inline libs/net/dal/Configuration/SourceConfiguration.cs:24]: According to your entity it is required.

Comment (laidaoyu) [reply libs/net/dal/Configuration/SourceConfiguration.cs]: I will update. thanks

File: libs/net/dal/Configuration/SeriesConfiguration.cs (hunk 3 above)
Comment (laidaoyu) [reply libs/net/dal/Configuration/SeriesConfiguration.cs]: I will update. thanks

File: libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql (hunk 2 above)
Comment (laidaoyu) [reply libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql:4]: so I don't need this PostUp at all?

File: libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql (hunk 1 above)
Comment (laidaoyu) [reply libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql:4]: so I don't need this script?

Comment (Fosol) [reply libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql:4]: Not unless you need to do more than add a column

Comment (laidaoyu) [reply libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql]: updated. thanks!

File: libs/net/dal/Configuration/SeriesConfiguration.cs (hunk 3 above)
Comment (laidaoyu) [reply libs/net/dal/Configuration/SeriesConfiguration.cs]: updated. thanks

File: libs/net/dal/Configuration/SourceConfiguration.cs (hunk 4 above)
Comment (laidaoyu) [reply libs/net/dal/Configuration/SourceConfiguration.cs]: updated. thanks

File: libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql (hunk 2 above)
Comment (laidaoyu) [reply libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql]: updated. thanks

**Review by Fosol — COMMENTED** [review]
Where did the DB migration go?

**Comment by Fosol** [comment]
Odd, when I reviewed last night it only showed 4 file changes in the PR.

**Review by Fosol — APPROVED** [review]
Don't forget to run the DB migration manually with the helper scripts.  Openshift can't handle the build of the db-migration service regrettably.

**Pull request merged**